livebooks/manage_the_zoo_with_pipes.livemd

```diff
@@ -0,0 +1,109 @@
+# Manage the zoo with pipes
+
+## Try it yourself!
+
+[![Run in Livebook](https://livebook.dev/badge/v1/black.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2Finspired-consulting%2FElixir-Meetup-Cologne%2Fblob%2Fmain%2Flivebooks%2Fmanage_the_zoo_with_pipes.livemd)
```

Comment:
I can't run the livebook 😕

![image](https://user-images.githubusercontent.com/78167297/225616144-afc8480b-9801-49d5-a1da-8fa76b126933.png)

Reply:
Because it was not merged yet :)